Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -848,7 +848,7 @@ script:
     - contextPath: Splunk.UserMapping.SplunkUser
       description: splunk user mapping.
       type: String
-  dockerimage: demisto/splunksdk:1.0.0.32388
+  dockerimage: demisto/splunksdk:1.0.0.24033
```

Comment:
i think because we have new docker image (32388) the build will fail
and we will need to create a new docker image with the specific good splunk SDK version
@yuvalbenshalom correct me if i'm wrong

Reply:
I created a new one